### PR TITLE
Remove intricate from CODEOWNERS

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # General reviewers per PR
-#                       David          Duncan   Jordan     Luke       Erik   John
-*                       @DavidEichmann @dcoutts @Jimbo4350 @intricate @erikd @newhoggy
+#                       David          Duncan   Jordan     Erik   John
+*                       @DavidEichmann @dcoutts @Jimbo4350 @erikd @newhoggy
```
Comment (erikd) [inline CODEOWNERS:3]: :sob:

**Review by erikd — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (Jimbo4350) [reply CODEOWNERS:3]: @intricate you're finally free :laughing:

Comment (intricate) [reply CODEOWNERS:3]: :rofl:

**Pull request merged**